index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Waleed's Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <header>
+            <nav class="nav-bar">
+            <div class="nav-bar-item logo">
+              <a href="#"><img class= "logo" src="./icons/logo.svg" alt="Logo" height="30" width="100"></a></div>
+            <div class="nav-bar-item">
+
+              <img class="menu" src="./icons/menu.png" alt="menu" height="25" width="25">
+              
+            </div>
+        </nav>
+        </header>
+        
+        <div class="landing-section">
+            
+            <section class="landing-text">
+                
+                <h1>Hey there. I’m <br>Waleed<br>
+                <span id="landingtext">I’m a software <br>developer</span>
```

Comment:
- [ ] The subtitle (if existing) uses a header tag `hx (examples: `h1`,`h2`,`h3`, etc...)` Kindly check that.

Reply:
I didn't understand what you are trying to say can you please elaborate? @aimemalaika